.github/workflows/build.yaml

```diff
@@ -206,12 +206,26 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
What's the purpose of this condition?

Reply:
This means the job is not checked by default in PRs. It's designed to do everything except actual push to a remote repo for PRs. This can stay the same except not making a release if it's not a tag job.

Reply:
You'd want it to push the branch for non-tag jobs?

Reply:
PRs by default don't have tags, only source branches.

Reply:
Yes, but if I do as you say and move this condition to the task that makes the release, all the other tasks would still run. Including, I think, the one that pushes a branch to gitlab (containing garbage at that point.)

So do you want that to happen or do you want this condition to be applied to the push as well?

Reply:
The goal is to run as much as possible for PRs so yes, the condition should be on the release step. Pushing to gitlab is fine. It will not by default if `PUSH_URL` variable is not specified or empty. And it will be just files with not working URLs which could be removed by deleting the branch but at least their contents can be checked.

Reply:
Done.

Reply:
Except of course now it fails because `20/merge` contains a slash...

Reply:
Okay, this probably works.